x-pack/plugins/lens/public/metric_visualization/visualization.tsx

```diff
@@ -115,4 +115,9 @@ export const metricVisualization: Visualization<State> = {
   removeDimension({ prevState }) {
     return { ...prevState, accessor: undefined };
   },
+
+  getErrorMessage(state, frame) {
+    // Is it possible to break it?
+    return undefined;
```

Comment:
I'm fine with requiring the function, but why `return undefined;`? What about no return statement?

Reply:
The type system has a distinction between `void` and `undefined` values:
```
 Type 'void' is not assignable to type '{ shortMessage: string; longMessage: string; } | undefined'.
```